README.md

```diff
@@ -295,6 +296,21 @@ in pkgs.poetry2nix.mkPoetryApplication {
 }
 ```
 
+
+## FAQ
+
+**Q.** I'm experiencing one of the following errors, what do I do?
+
+- ModuleNotFoundError: No module named 'setuptools'
+- ModuleNotFoundError: No module named 'pdm'
+- ModuleNotFoundError: No module named 'setuptools-scm'
+- ModuleNotFoundError: No module named 'poetry-core'
+- ModuleNotFoundError: No module named 'flit'
+- ModuleNotFoundError: No module named 'flit-core'
+- ModuleNotFoundError: No module named 'flit-scm'
+
+**A.** Have a look at the following document [edgecase.md](./docs/edgecases.md)
```

Comment:
Would it be better to inline the single edge case here rather than keeping it as a separate document? I feel like it's common enough (a large percentage of Python package updates break because of a missing build tool) that it warrants being more highlighted.

Reply:
IMO the document is too large to be merged into the README and I feel it really requires the detailed explanation.